test/extended/builds/sti-build-no-outputname.go

```diff
@@ -0,0 +1,39 @@
+package builds
```

Comment:
can these different build types be wrapped into a single test file?  See sample_repos.go for example.